custom/icds/data_management/static/data_management/js/data_management.js

```diff
@@ -1,9 +1,94 @@
 hqDefine('data_management/js/data_management', [
     'jquery',
+    'knockout',
+    'underscore',
+    "hqwebapp/js/initial_page_data",
+    'hqwebapp/js/components.ko',    // pagination widget
+    'hqwebapp/js/knockout_bindings.ko', // for modals
     'jquery-ui/ui/datepicker',
-], function ($) {
+], function ($, ko, _, initialPageData) {
     'use strict';
+
+    var dataManagementRequest = function (name, dbAlias, createdAt, initiatedBy, startedOn, endedOn, startDate,
+        endDate, status, statusText, error) {
+        var self = {};
+        self.name = name;
+        self.dbAlias = dbAlias;
+        self.initiatedBy = initiatedBy + gettext(' On ') + createdAt;
```

Comment:
This should use `_.template` to allow translators to reorder the variables if needed. `./corehq/apps/accounting/static/accounting/js/pricing_table.js` has examples of this technique.